ai.menlo.jan.yml

```diff
@@ -0,0 +1,62 @@
+app-id: ai.menlo.jan
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+
+command: run.sh
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --env=ELECTRON_TRASH=gio
+  - --env=NVIDIA_VISIBLE_DEVICES=all
+
+modules:
+  - name: jan
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin /app/jan /app/tmp-install/data
+      - bsdtar -xf jan-linux-amd64-0.5.12.deb -C /app/tmp-install
+      - bsdtar -xf /app/tmp-install/data.tar.xz -C /app/tmp-install/data
+      - mv /app/tmp-install/data/opt/Jan/* /app/jan/
+      - rm -rf /app/tmp-install jan-linux-amd64-0.5.12.deb
+      - install -Dm755 -t /app/bin/ run.sh
+      - install -Dm644 -t /app/share/applications/ ai.menlo.jan.desktop
+      - install -Dm644 -t /app/share/metainfo/ ai.menlo.jan.metainfo.xml
+      - install -Dm644 ai.menlo.jan.png /app/share/icons/hicolor/256x256/apps/ai.menlo.jan.png
+    sources:
+      - type: file
+        url: https://github.com/janhq/jan/releases/download/v0.5.12/jan-linux-amd64-0.5.12.deb
+        dest-filename: jan-linux-amd64-0.5.12.deb
+        sha256: eb4d4694816fd967d54addb43fe0212d9e9d10cde6bd2c41d12754448404c7e9
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/jan/jan "$@"
+      - type: file
+        path: ai.menlo.jan.desktop
+      - type: file
+        path: ai.menlo.jan.metainfo.xml
+      - type: file
+        path: ai.menlo.jan.png
+
+cleanup:
```

Comment:
this section should be above `modules`

Reply:
also are these things really copied and in need of being removed?

I don't see where.